.github/workflows/lucide-angular.yml

```diff
@@ -45,4 +45,3 @@ jobs:
 
       - name: Test
         run: pnpm --filter lucide-angular test
-
```

Comment:
Again, great job. Only thing I'm not sure about is removing the new lines at the end of files, if anything I would've added a new line to release.yml. @ericfennis? 🤔

Reply:
I have a setting on to trim unnecessary whitespace. Can roll that back manually if you want!

Reply:
I think it's fine, I checked we have still a EOF.